gtsfm/utils/transforms.py

```diff
@@ -0,0 +1,48 @@
+"""Utility functions for geometric transformations.
+
```

Comment:
utils/spherical.py is too restrictive, I'd like to put transforms to other coordinate frames here as well. Let me know if there is a util file for that already. coordinate_systems is kind of okay, but it sounds more like the definition of a coordinate system rather than transforming between them. coordinate_transforms or coordinate_conversions might be better.

Reply:
`coordinate_conversions.py` seems good to me.